Umweltzone/src/main/res/values-fr/ckchangelog.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+    <string name="changelog_title">Ce qui a changé</string>
+    <string name="changelog_show_full">Voir plus &#8230;</string>
+    <string name="changelog_version_format">
+        Version <xliff:g id="version_name">%s</xliff:g>
+    </string>
+    <string name="changelog_error_resource_not_android_5">"        Erreur lors de l&#8217;affichage des informations sur la version   "</string>
```

Comment:
Please remove the wrapping double quotes (`"`) and the unneeded whitespace. These cause formatting issues.